src/style.css

```diff
@@ -0,0 +1,88 @@
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+.search_bar {
+  position: relative;
+  margin-top: 20px;
+  display: flex;
+  width: 100%;
+}
+
+.search_input {
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  font-family: inherit;
+  font-style: inherit;
+}
+
+.search_input_big {
+  height: 70px;
+  padding-left: 62px;
+  line-height: 22px;
+  background-size: 19px;
+  background-position: 26px 25px;
+  font-size: 16px;
+}
+
+.search_input_small {
+  height: 42px;
+  padding-left: 33px;
+  line-height: 11px;
+  background-size: 11px;
+  background-position: 13px 15px;
+  font-size: 14px;
+}
+
+.search_input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  font-family: inherit;
+}
+
+.search_input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  outline: none;
+  font-family: inherit ;
+  background-repeat: no-repeat;
+  background-image: url("images/Search.svg");
+}
+
+.search_input_big::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
```

Comment:
Use CSS variables for all repeatable colors and move them :root selector.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)

Reply:
Hello , something like that :root {
  --main-bg-color: brown;
}
?

Reply:
.search_input::placeholder {
  color: #3d4e61;
  font-family: inherit;
}

.search_input_big::placeholder {
  font-size: 16px;
  line-height: 22px;

}

.search_iput_small::placeholder {
  font-weight: 400;
  font-size: 14px;
  line-height: 19px;
}
?